build.gradle

```diff
@@ -77,6 +77,7 @@ dependencies {
     compile 'org.antlr:ST4:4.0.8'
     compile 'org.glassfish.jersey.core:jersey-client:2.22.2'
     compile 'net.jcip:jcip-annotations:1.0'
+    compile 'org.objenesis:objenesis:2.2'
```

Comment:
Required to create [_lazily initializable proxy_](https://ci.apache.org/projects/wicket/apidocs/6.x/org/apache/wicket/proxy/LazyInitProxyFactory.html) for Wicket component depencies that do not have no args constructors (usually Spring beans with constructor dependency injection).